src/main/java/net/tammon/sip/packets/ReadOnlyData.java

```diff
@@ -27,13 +27,13 @@
 
 public class ReadOnlyData extends AbstractPacket implements Request {
 
-    private static final int messageType = 71;
+    public static final int MSG_READ_ONLY_DATA = 71;
```

Comment:
Be consistent in naming of message type
```suggestion
    public static final int MSG_TYPE_ID = 71;
```